readme.md

```diff
@@ -53,98 +50,64 @@ Broken into a variety of functional areas, the eligibility criteria is a checkli
 
 **Suitability**
 
--The project operates within moral and ethical standards of the .NET Foundation.
-
--The project is aligned with the philosophy and guidelines for collaborative development.
-
--The project is built on the .NET platform and/or creates value within the .NET ecosystem.
-
--The project solves a large enough problem and has a clear technology scope and focus which provides a reasonable likelihood of success.
+- The project operates within moral and ethical standards of the .NET Foundation.
+- The project is aligned with the philosophy and guidelines for collaborative development.
+- The project is built on the .NET platform and/or creates value within the .NET ecosystem.
+- The project solves a large enough problem and has a clear technology scope and focus which provides a reasonable likelihood of success.
 
 **Code**
 
--The project produces software for distribution to the public at no charge.
-
--The project's code is easily discoverable and publicly accessible (preferably on GitHub). 
-
--The project contains a build script that can produce deployable artifacts that are identical to the official deployable artifacts, with the exception of code signing (Exception may be granted for strong name keys, though strongly encouraged to be committed. Exception relies on OSS signing being in the build script for public builds).
-
--When applicable, project must use reproducible build settings in its toolchain.
-
--The project uses [Source Link](https://docs.microsoft.com/en-us/dotnet/standard/library-guidance/sourcelink).
-
--The project uses either embedded PDBs or publish symbol packages to NuGet (if applicable).
-
--The project code signs their artifacts as appropriate.
+- The project produces software for distribution to the public at no charge.
```

Comment:
Still have concerns about this. I believe the source code should be distributed at no charge per the[ Open Source Definition](https://opensource.org/osd-annotated), but we shouldn't require the software itself be free of charge. People should be able to charge for the end user software.

Reply:
I also have concerns and believe the OSD isn't good enough. Thoughts:

- projects should be free to charge for distribution but must accept contributions from the community at the discretion of the project maintainers.


Ie "you are free to use the software, if the licensed allows (this against OSD) but ultimately we want to encourage you to engage with the project how it built in an open manner"

Ie. "Don't over index on the rights of the consumer"

Reply:
> you are free to use the software, if the license allows (this against OSD)

I don't understand how this is against the OSD. Could you elaborate?

> Projects do not need to be use a OSD blessed license.

To be pedantic, the OSD is a definition that the open source community by and large accepts for a license to be considered open source. The OSI (Open Source Initiative) approves (and rejects) licenses.

So to restate your point, it seems you're suggesting that projects do not need to use a license that meets the Open Source Definition.

If that captures what you're suggesting, I would ask why should we allow projects to use a license that does not meet the OSD? I don't think the .NET Foundation wants to be in the business of defining its own open source definition. We don't have to accept _every_ license that meets the OSD. We could decide to only accept a subset. We could add extra requirements for .NET Foundation projects.

But I think the OSD is a good starting point unless there's some really compelling reason to diverge.

Reply:
> projects should be free to charge for distribution but must accept contributions from the community at the discretion of the project maintainers who have a responsibility to act in the best interests of the project (which may just mean acting in the interests of the maintainers!)

This seems to contradict itself to be a non-statement. They __MUST__ accept contributions __AT THEIR DISCRETION__. Which is it? Must they, or do they have discretion?

In terms of evaluating a project, how do we evaluate this? Do we require that a project has accepted contributions from non-maintainers? What about projects that add all contributors to maintainers (some do this now).

Reply:
@haacked I agree the OSD is a good starting point but don't agree that restricting scope of foundation to "opensource projects."

See @dependabot as a reference. It is NOT opensource as by the OSD. Yet it's openly developed. That's the situ I want to see more of.

What they are doing is totally valid and amazing but "is not opensource".

Look at all those PRs from the community fixing their own problems.

https://t.co/BiQQjFz8NQ

It hasn't been defined yet afaik . I'm calling it "openly developed software" - a wrapper around opensource that enables maintainers or projects to not be bound by the restrictions of OSI if they so choose. Key principles is openly developed.


That's what I'd like to see enabled and possible in the @dotnetfdn. It's all super new tho. Let's take this convo off GitHub as I don't want to distract folks. See twitter DM.

Reply:
Even though we both have very similar _thoughts_ on open software, I'm going to deviate from @ghuntley a bit here. I'm a big fan of supporting and normalizing "openly developed" software as an alternative to "open source" (conceding "open source" == OSI-approved license). There are lots of good reasons why a maintainer might choose to go that route and it should be their choice to make.

That said, I actually think it's okay for the Foundation to take the position that "openly developed software is cool and all, but that's just not our wheelhouse for projects inside the .NET Foundation." It _doesn't_ mean that the Foundation can't support, promote, or even use (I.e. Dependabot) such projects in other ways and I would hope that it does, just that those projects probably aren't suitable to be _member projects_.

...which opens up a bigger question for me about "what really is a member project?" - I'll open a separate comment for that.